cola/libdialect/util.h

```diff
@@ -94,7 +94,7 @@ struct Matrix2d {
     std::string toString() const {
         std::string s = "";
         s += "\n  ";
-        char buffer [10];
+        char buffer [12];
```

Comment:
Maybe 21 would be better? My C++ book says that an `int` has a "minimum size" of 16 bits, whereas this compiler evidently was using 32 bits, since it claimed it could take up to 10 decimal places to express an integer. Could there be some weird architecture out there that used 64-bit ints? I don't know. If so, they'd want 20 decimal places, and then there's the nul terminal char.

Meanwhile it's kind of funny that the compiler bothers to complain about this, while it lets us get away with line 107, where we use the same buffer to print an *element* of the matrix, which is of unspecified type `T`. That could turn out to be `long long` in some application, so again maybe 21 is better. And it's not like it's any big cost.

Reply:
According to https://stackoverflow.com/questions/589575/what-does-the-c-standard-state-the-size-of-int-long-type-to-be

The minimum size of int, long, long long is guaranty but not the maximum size.

Reply:
Okay, yeah, that was my sense of it, so might as well use a generous buffer and forestall having to address this again any time soon.